app/vmui/Dockerfile-web

```diff
@@ -6,7 +6,7 @@ COPY web/ /build/
 RUN GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build -o web-amd64 github.com/VictoriMetrics/vmui/ && \
     GOOS=windows GOARCH=amd64 CGO_ENABLED=0 go build -o web-windows github.com/VictoriMetrics/vmui/
 
-FROM alpine:3.17.3
+FROM alpine:3
```

Comment:
This change looks incorrect, since it tries updating the tag for the exact Alpine release to the tag, which can point to different Docker images over time.

Also, I couldn't find the relevant information about the vulnerability on the [Alpine releases](https://www.alpinelinux.org/releases/) page - the latest stable `3.17.x` release is `3.17.3` according to this page, while the [3.18.0 release](https://www.alpinelinux.org/posts/Alpine-3.18.0-released.html) doesn't mention the reported security issue.